pages/profileOfSomeoneElse.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { requireAuth } from "../components/auth"
+import * as links from "../components/links"
+import { createPage } from "../components/page"
+import SelectLegislators from "../components/SelectLegislators"
+import MyTestimonies from "../components/MyTestimonies/MyTestimonies"
+
+export default createPage({
```

Comment:
consider leaving the public profile page to another PR, since we'll need to modify this significantly to show an arbitrary user's profile